crates/viewer/re_renderer/src/importer/stl.rs

```diff
@@ -21,11 +22,21 @@ pub enum StlImportError {
 
 /// Load a [STL .stl file](https://en.wikipedia.org/wiki/STL_(file_format)) into the mesh manager.
 pub fn load_stl_from_buffer(
-    buffer: &[u8],
+    asset3d: &Asset3D,
```

Comment:
as mentioned above, this type shouldn't be here. Also why does it need to be and why do the other loaders don't have them